Remotion/ObjectBinding/Web.ClientScript/Scripts/BocList.ts

```diff
@@ -167,24 +168,28 @@ class BocList
       }
     });
   
-    $(selectorControl).click(function (evt)
+    selectorControl.addEventListener('click', function (evt)
     {
       evt.stopPropagation();
       BocList.OnRowSelectorClick();
-      $(row).trigger ('click');
+      row.dispatchEvent (new MouseEvent('click'));
     });
   
       // Enable the entire selector control's cell for click events that mimic the selector control.
-    $(cell).click(function (evt)
+    cell.addEventListener('click', function (evt)
     {
       BocList.OnRowSelectorClick();
     });
   
     // Enable the entire row to the left of the selector control's cell for click events that mimic the selector control.
-    $(cell).prevAll().click(function (evt)
+    let prev: Nullable<Node> = cell;
+    while (prev = prev.previousSibling)
```

Comment:
the loop-variable should perhaps not be 'prev' / 'previous' but 'current'. That we iterate the previous items is declared via 'previousSibling', but making 'cell' the 'prev' just feels strange.